src/average.ts

```diff
@@ -1,7 +1,7 @@
-export const average = (inputs: any) => {
+export const average = (inputs: number[]) => {
     var sum = 0;
```

Comment:
Use `const` and `let` instead of `var`

If it's a variable that you aren't going to reassign, use `const`. If you are going to reassign it, use `let`

Always try `const` first, if you get an error then use `let`